packages/run-protocol/src/vpool-xyk-amm/addPool.js

```diff
@@ -41,8 +41,9 @@ export const makeAddIssuer = (zcf, isInSecondaries, brandToLiquidityMint) => {
         AssetKind.NAT,
         harden({ decimalPlaces: 6 }),
       ),
-      mint => {
-        zcf.saveIssuer(secondaryIssuer, keyword);
+      /** @param {ZCFMint} mint */
+      async mint => {
+        await zcf.saveIssuer(secondaryIssuer, keyword);
```

Comment:
noting https://github.com/Agoric/agoric-sdk/issues/5407 to link this instance to that ticket